app/scripts/proactive/view/xml/JobXmlView.js

```diff
@@ -42,7 +42,43 @@ define(
             // removing multiple \n before closing xml element tag
             jobRendering = jobRendering.replace(/\n+\s+>/g, '>\n');
             // indenting using vkbeautify
-            return vkbeautify.xml(jobRendering.trim(), 2);
+            return this.beautifyAndPreserveCDATA(jobRendering.trim(), 2);
+        },
+        // pretty-print xml and preserve CDATA sections from indentation changes
+        beautifyAndPreserveCDATA: function(text, beautifyDepth) {
+            var that = this;
```

Comment:
where/how is this used `var that = this;` ?

Reply:
this is typical javascript code, even if not used maybe now in this function, it is the general way to allow a subfunction call accessing the "this" object